pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1058,7 +1058,9 @@ public PulsarClient getReplicationClient(String cluster) {
                         .enableTcpNoDelay(false)
                         .connectionsPerBroker(pulsar.getConfiguration().getReplicationConnectionsPerBroker())
                         .statsInterval(0, TimeUnit.SECONDS);
-                if (pulsar.getConfiguration().isAuthenticationEnabled()) {
+                if (data.getAuthenticationPlugin() != null && data.getAuthenticationParameters() != null) {
```

Comment:
Can this feature be disabled?

Reply:
If you don't set authPlugin and authParameters, it will be the existing behavior, no?

Reply:
But we now enable and disable Authentication through isAuthenticationEnabled...

Reply:
I just refer AuthenticationConfig in function, which is used when creating pulsar client in function runner, and there is no isAuthenticationEnabled setting. Although it is easy to add this setting if needed. But user should make sure not forget to set this parameter with true when create remote cluster with admin cli tool.

Reply:
OK, thanks for your explanation